src/module/vue/components/buttons/btn-rollmove.vue

```diff
@@ -12,25 +12,43 @@
 </template>
 
 <script setup lang="ts">
+import { computed } from '@vue/reactivity'
 import { inject } from 'vue'
+import { Move } from '../../../features/custommoves.js'
 import { IronswornPrerollDialog } from '../../../rolls'
 import { $ActorKey } from '../../provisions'
 import btnIsicon from './btn-isicon.vue'
 
 const props = defineProps<{
-  move?: any
-  tooltip?: string
+  move?: Move
   disabled?: boolean
 }>()
 
+const tooltip = computed(() => {
+  let str = game.i18n.format('IRONSWORN.RollMove', {
+    title: props.move?.displayName,
+  })
+  // const { Title, Page } = props.move?.dataforgedMove?.Source ?? {}
```

Comment:
Still deciding on which of these to use? It'd be _nice_ to use the page numbers somewhere, but it's not easy to find a good place for them.

Reply:
> Still deciding on which of these to use? It'd be _nice_ to use the page numbers somewhere, but it's not easy to find a good place for them.

yeah. with the introduction of PDF JEPs, there's maybe even a direct use case for them (maybe even the possibility of direct links if the API exposes a way to point to a specific page)... but that's *definitely* a different PR.

for now: maybe some kind of secondary text in the move rules display? like a footer with something like "Source: *Ironsworn Rulebook*, p. XX"

Reply:
on second thought -- that adding a footer to the move text rendering could be its own PR, if you consider losing the `title` page numbering acceptable change in the interim. i'll remove the comment here and add an appropriate TODO elsewhere to note it (and then start the PR to add it).

Reply:
Yeah, that's fine. I think the way the code works now provides the most useful tooltip for where it is.

Reply:
To clarify: maybe delete the commented-out code here, and this should be ready to merge?